src/components/ValidationTable/ValidationTable.jsx

```diff
@@ -123,36 +133,62 @@ const ValidationTable = ({ data }) => {
     setColumnsWithClickableNames(auxData);
     setAuxColumnsWithClickableNames(auxData);
   // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  }, [data?.length]);
+
+  const currentValidationRecord = useSelector((state) => state.currentValidationRecord);
+  const courseSelected = useSelector((state) => state.currentValidationRecord);
 
   return (
-    <DataTable
-      isFilterable
-      defaultColumnValues={{ Filter: TextFilter }}
-      itemCount={data.length}
-      data={adaptToTableFormat(data)}
-      columns={columnsWithClickableNames}
-      additionalColumns={!isValidator ? [
-        {
-          id: 'action',
-          Header: 'Action',
-          Cell: ({ row }) => {
-            const label = ActionsAvailable[row.values.status?.toLowerCase()]?.label || '';
-            const action = ActionsAvailable[row.values.status?.toLowerCase()]?.action || '';
-            return label ? <ActionButton label={label} action={() => action(row)} /> : null;
+    <>
+      <ModalLayout
+        isOpen={isOpen}
+        onClose={close}
+        tabs={[
+          {
+            name: 'validation_process',
+            label: 'Validation process',
+            component: <ValidationProcess courseSelected={courseSelected} />,
```

Comment:
If you are sending the same information, I don't think we need different reference to the seme object.
```suggestion
            component: <ValidationProcess courseSelected={currentValidationRecord} />,
```